external-api/src/http/external_match.rs

```diff
@@ -185,6 +185,20 @@ impl ExternalOrder {
         let div = FixedPoint::floor_div_int(min_fill, price) + Scalar::one();
         scalar_to_u128(&div)
     }
+
+    /// Get the quote amount of the order implied by the external order
+    ///
+    /// The price here is expected to be decimal corrected; i.e. multiplied by
+    /// the decimal diff for the two tokens
+    pub fn get_quote_amount(&self, price: FixedPoint) -> Amount {
```

Comment:
will you move this to be co-located with `get_base_amount` above, and make `get_base_amount` public?